[Android] ARM64: Split Android.mk file

leemgs opened the pull request:
This commit is to split the existing Android.mk file in order to maintain
(1) nnstreamer library (e.g., libnnstreamer.so) and (2) test applications
separately.

* Before: Android.mk
* After :
  - Android-nnstreamer.mk (to generate nnstreamer library)
  - Android-app.mk (to generate nnstreamer-based test application)
---

leemgs commented:
* How to build
1. nnstreamer library 
rm -rf ../libs/ ../obj/  ; ndk-build APP_BUILD_SCRIPT=./Android.mk -j$(nproc)
cp ../libs/arm64-v8a/libnnstreamer.so $GSTREAMER_ROOT_ANDROID/arm64/lib

2. test application 
ndk-build APP_BUILD_SCRIPT=./Android-app.mk

jijoongmoon commented on jni/Android-app.mk:
```diff
@@ -0,0 +1,382 @@
+LOCAL_MODULE := gstreamer-1.0
+LOCAL_SRC_FILES := $(GSTREAMER_ROOT)/lib/libgstreamer-1.0.so
+include $(PREBUILT_SHARED_LIBRARY)
+
```
How about using foreach and define to reduce code length?

leemgs replied:
Thanks. PR will be updated.

leemgs commented:
**Changelog**

* Version 2:
1. Added two functions to simplify duplicated statements with foreach (by @jijoongmoon  comment)
2. Updated the existing comment to build C/C++ source code with ndk-build

* Version 1:
1. Added Android-nnstremaer.mk
2. Added Android-app.mk
